doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/configuring-autoscaling.rst

```diff
@@ -2,55 +2,44 @@
 
 .. _deployment-guide-autoscaler-under-construction:
 
-Autoscaling with Ray
---------------------
+Configuring the Ray Autoscaler
+==============================
 
-Ray is designed to support highly elastic workloads which are most efficient on
-an autoscaling cluster. At a high level, the autoscaler attempts to
-launch/terminate nodes in order to ensure that workloads have sufficient
-resources to run, while minimizing the idle resources.
+This page provides an overview of the parameters for the Ray autoscaler. A Ray cluster by default runs an autoscaler that modifies
+the number of nodes based on the resources requested by the task, actor, or placement group. If there is insufficient resource to serve
+the request they will be put into pending and will not execute. The autoscaler runs a bin-packing algorithm to determine the list of nodes
+the cluster should use to satisfy the resource requests, and adds / removes nodes from the cluster to maximize the cluster utilization
+while providing sufficient resource to execute the requests. The autoscaler currently only looks at the amount of cpu and gpu requested
+and adds / removes nodes accordingly. It does not take into the account the memory request nor the disk usage.
 
-It does this by taking into consideration:
+Parameters
+==========
 
-* User specified hard limits (min/max workers).
-* User specified node types (nodes in a Ray cluster do _not_ have to be
-  homogenous).
-* Information from the Ray core's scheduling layer about the current resource
-  usage/demands of the cluster.
-* Programmatic autoscaling hints.
+**max_workers[default_value=2, min_value=0]**: Indicates the maximum number of workers the cluster will have at any given time. This number
+excludes the head node where there is always exactly one.
 
-Take a look at :ref:`the cluster reference <cluster-config>` to learn more
-about configuring the autoscaler.
+.. note::
+  When this value changes and drops below the number of running workers, the autoscaler will remove nodes to satisfy the new constraint even
+  if the node is running some workload. Workers that are the least recently used will be removed first.
 
+**upscaling_speed[default_value=1.0, min_value=0.0]**: The number of nodes allowed to be pending as a multiple of the current number of nodes.
```

Comment:
min_value should be 1 https://github.com/ray-project/ray/issues/24177

Reply:
sg